clients/web/src/lib/components/organisms/scrims/AvailableAdminView.svelte

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+    import {
+        AdminScrimTable, DashboardLayout, DashboardCard,
+    } from "$lib/components";
+import AdminPlayerTable from "../../molecules/scrims/AdminPlayerTable.svelte";
+
+    // Is this visible?
+    export let visible = true;
+
+    // State Machine of badness
+    export let adminScrimManagementVisible = false;
+    export let adminPlayerManagementVisible = false;
+    export let adminLeagueManagementVisible = false;
```

Comment:
Just change this to `selectedView: "scrim | player | league"`, so we don't need 3 booleans